src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,77 @@
-export const Pagination = () => {};
+import React from 'react';
+
+interface PaginationProps {
+  total: number;
+  currentPage: number;
+  perPage: number;
+  onPageChange: (pageNumber: number) => void;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  total = 42,
```

Comment:
```suggestion
  total,
```
It isn't an optional props